app/src/main/res/layout/list_stream_item.xml

```diff
@@ -75,6 +75,7 @@
         android:layout_alignParentBottom="true"
         android:layout_toRightOf="@+id/itemThumbnailView"
         android:layout_toEndOf="@+id/itemThumbnailView"
+        android:ellipsize="end"
```

Comment:
This should be applied to all list stream items: local, grid, mini. (I'm not really sure though)

Reply:
The additional details view only exist in this variant (the complete one). I think the other text views already have this set.